### DISCUSSION TITLE
Enchanted books won't merge

### DISCUSSION THREAD

**Discussion by Seeelvan** [Geolykt/EnchantmentsPlus#118] [Q&A]
I am using Aternos and Spigot 1.18.1 for this. I already tried enabling the anvil-merger in the patches.yml and it still doesn't work. Please help if I did this right.
![image](https://user-images.githubusercontent.com/100981067/156864416-1e6ff783-9dd7-43a9-8a7c-7d42a9d67cd1.png)

**Comment by Geolykt** [comment]
anvil-merger needs to be `"1xx"` on spigot, not `true "4xx"`. But honestly, don't use spigot - and if you are using it anyways then it is because of the yank behaviour - so working as intended more or less.

That being said, if SOMEONE is able to find some (even if hacky) workaround to this whole situation - then they are free to let me know

**Reply by Seeelvan** [reply]
okay thanks I got it to work now

**Reply by Seeelvan** [reply]
Also, can I edit the effects of enchantments? I'd like to lessen the effects of some of it to balance with my server.

**Reply by Geolykt** [reply]
Yes in the world.yml config file (don't forget to also do it in the world_end.yml and world_nether.yml) you can change the "power" of each enchantment.

**Reply by Seeelvan** [reply]
Okay thanks again. BTW, amazing plugin to spice up the world
